ncm-accounts/src/main/pan/components/accounts/sysgroups.pan

```diff
@@ -93,9 +93,9 @@ unique template components/accounts/sysgroups;
     'webalizer', '',
     'cimsrvr', '',
     'vboxusers', '',
-    'apacheds', '',
-    'mongod' , '',
-    'mongodb' , '',
-    'rabbitmq' , '',
-    'memcached' , '',
-   );
+    'apacheds', '',
+    'mongod' , '',
```

Comment:
looks like a space before the comma is inserted. is this a bug in the script or in the github diff viewer?